src/plugins/orangehrmCoreOAuthPlugin/Dto/OAuthClientSearchFilterParams.php

```diff
@@ -23,10 +23,10 @@
 
 class OAuthClientSearchFilterParams extends FilterParams
 {
-    public const ALLOWED_SORT_FIELDS = ['oc.clientId'];
+    public const ALLOWED_SORT_FIELDS = ['oAuthClient.id'];
```

Comment:
```suggestion
    public const ALLOWED_SORT_FIELDS = ['oauthClient.id'];
```
I prefer `oauthClient.id` or `client.id`